recipes/diligentgraphics-spirv-tools/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,23 @@
+cmake_minimum_required(VERSION 3.1)
+project(test_package)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup(TARGETS)
+
+find_package(SPIRV-Tools REQUIRED CONFIG)
+
+add_executable(${PROJECT_NAME}_c test_package.c)
+if(TARGET SPIRV-Tools-shared)
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-shared)
+elseif(TARGET SPIRV-Tools-static)
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-static)
+else()
+    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools)
+endif()
```

The only reason this is failing I can think of, is that the target `SPIRV-Tools` does not exist.
If no target named `SPIRV-Tools` exists, then cmake will interpret it as a library and only uses it at link time.

Since this recipe is a fork of the `spirv-tools` recipe, I don't know why it would fail here and not in `spirv-tools`.

```suggestion
if(TARGET SPIRV-Tools-shared)
    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-shared)
elseif(TARGET SPIRV-Tools-static)
    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools-static)
else()
    if(NOT TARGET SPIRV-Tools)
        message(FATAL_ERROR "There is no SPIRV-Tools target")
    endif()
    target_link_libraries(${PROJECT_NAME}_c SPIRV-Tools)
endif()
```